src/types.ts

```diff
@@ -82,9 +83,22 @@ export type Direction =
   | 'BottomLeft'
   | 'BottomRight';
 
+export type TransformType = 'rotate'
+
+export type TransformProperty = {
+  [key in TransformType]: string | number | boolean | Style;
+}
+
+export type TransformStyle = {
```

Comment:
this might be necessary, but I'm also wondering why the need for a new style ir type. Could the whole concept work as a "dependent" style instead?  I haven't dug as deep into as you have, just thinking aloud...

Reply:
reading quickly what dependent does, is that it yields a style computed from another style. Here i believe we _could_ do that, by aggregating the values(not merged into an object, but concatenated in an array). 
I guess the only thing i'll need to test, is to see if dependent has access to itself.

To paraphrase the letter-spacing example, this is what we would need here:

```
function aggregateTransforms(transformStyle: { [key in TransformProperty]: number | string }): DependentStyle {
  return {
    kind: `dependent`,
    complete(style) {
      const pastTransform = style.transform || [];
      style.transform = [...style.transform, transformStyle];
    },
  };
}

```